docs/pages/desktop-access/introduction.mdx

```diff
@@ -1,44 +1,72 @@
 ---
-title: Desktop Access
-description: Teleport desktop access introduction and resources.
+title: Manage Windows resources
```

Comment:
I get that we don't want to say "Desktop Access" here, but I think the word "access" is important. The whole point of this feature is to be able to **access** Windows infrastructure.